File: java-selenide-gradle-allure/src/test/java/TestClass1.java
import io.qameta.allure.Description;
import org.junit.jupiter.api.Test;

import static com.codeborne.selenide.CollectionCondition.*;
import static com.codeborne.selenide.Condition.text;
import static com.codeborne.selenide.Selectors.by;
import static com.codeborne.selenide.Selenide.$;
import static com.codeborne.selenide.Selenide.$$;

public class TestClass1 extends BaseTest {



    @Test
    @Description("Ololo")

    void t1(){
        $(by("name","q")).setValue("allure-nunit").pressEnter();
        $$(".repo-list li h3").filterBy(text("unickq/allure-nunit")).shouldBe(size(1));
    }

}

File: kotlin-selenide-gradle-allure/src/test/kotlin/TestClass1.kt
import com.codeborne.selenide.Condition
import com.codeborne.selenide.Selenide.`$$`
import com.codeborne.selenide.Selenide.`$`
import io.qameta.allure.Feature
import org.junit.jupiter.api.Test
import org.junit.jupiter.api.TestInstance
import org.openqa.selenium.By


@TestInstance(TestInstance.Lifecycle.PER_CLASS)
class TestClass1 : BaseTest() {

    @Test
    @Feature("Search")
    fun testPass() {
        `$`(By.name("q")).setValue("allure-nunit").pressEnter()
        `$$`(".repo-list li h3").filterBy(Condition.text("unickq/allure-nunit")).shouldHaveSize(1)
    }
}

